packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer-card.jsx

```diff
@@ -117,8 +117,8 @@ const ContractDrawerCard = ({
         >
             <div
                 className={classNames('dc-contract-card', {
-                    'dc-contract-card--green': is_mobile && !is_multiplier && profit > 0 && !result,
-                    'dc-contract-card--red': is_mobile && !is_multiplier && profit < 0 && !result,
```

Comment:
wondering why these checks are removed

Reply:
Same here. Gradient should be applied to all trade types, for ongoing/closed contracts, for desktop/responsive